docs/tutorials/exploring-datasets/movie-recommendation.md

```diff
@@ -179,31 +179,31 @@ ratings to the same movies. For the target user and some other user we
 are searching for the same movies:
 
 ```cypher
-MATCH (u:User {id:1000})-[r:RATED]-(m:Movie)-[other_r:RATED]-(other:User);
+MATCH (u:User {id:1000})-[r:RATED]-(m:Movie)-[other_r:RATED]-(other:User)
+RETURN *;
 ```
-
+If you try to execute the query above with added `RETURN` statement, you will get all
+potential similar users and movies they rated. 
 But this is not enough for finding similar users. We need to choose users with
 the same movies and similar ratings:
 
 ```cypher
 WITH other.id AS other_id,
      avg(abs(r.rating-other_r.rating)) AS similarity,
-     count(*) AS similar_user_count
-WHERE similar_user_count > 2
+     count(*) AS same_movies_rated
+WHERE same_movies_rated > 2
 WITH other_id
 ORDER BY similarity
 LIMIT 10;
+WITH collect(other_id) AS similar_user_set
 ```
 
-Here we calculate similarities as the average distance between target user rating
+Here we calculate similarities as the average distance between the target user rating
 and some other user rating on the same set of movies. There are two parameters:
-similarUserCount limit (2) and similar user set size limit (10). Similar user
-count limit is used for filtering users who have at least 2 movies in common
-with the target user. Similar user set size is used to peek top 10 similar users
-(10 or less).
+`same_movies_rated` defines the number of same movies (more than 3) that the target user and other users need to rate, and `similar_user_set` represents the users that gave a similar rating to the movies that the target user has rated. This parameters enable extracting best users for movie recomendations.
 
-Now we have similar user set. We will use those users to calculate the average
-score for all movies in the database.
+Now we have a similar user set. We will use those users to calculate the average
+rating score for all movies they rated in the database, and return the best rated that are ordered by rating quality, but as `prediction_score` variable.
```

Comment:
rating score or rating? Also, would it be better to change `prediction_score` to `prediction_rating` to be consistent?